app/src/main/java/com/teamltt/carcare/database/contract/ReminderContract.java

```diff
@@ -0,0 +1,140 @@
+/*
+ *
+ *  * Copyright 2017, Team LTT
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package com.teamltt.carcare.database.contract;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import com.teamltt.carcare.model.Reminder;
+
+public class ReminderContract {
+
+    public static final String SQL_CREATE_ENTRIES = "CREATE TABLE " + ReminderEntry.TABLE_NAME + " ("
+            // vehicle_id INTEGER PRIMARY KEY
+            + ReminderEntry.COLUMN_NAME_ID + " INTEGER PRIMARY KEY AUTOINCREMENT,"
+            + ReminderEntry.COLUMN_NAME_VEHICLE_ID + " INTEGER,"
```

Comment:
If this is a foreign key then the foreign key constraint needs to be added to this query string. See `OwnershipContract` for an example of how it should be added with cascades.